Add info about --global config options for Astro CLI (#3095)
astronomer/docs: astro/cli/reference.md

## Global Options

### Global flags

The Astro CLI has the following global flags that can be used with any command:

- `-h`, `--help`: Output more information about a given command to the CLI.
- `--verbosity <string>`: Specify the log level to expose for each CLI command. Possible values are `debug`, `info`, `warn`, `error`, `fatal`, and `panic`.

### Global configuration options

> - `g`, `--global`: You can globally configure certain properties for your Astro CLI by using this flag with the `astro config` command. See [Configure the CLI](configure-cli.md#global-configuration-options) for more information.

Reviewer: ```suggestion
- `g`, `--global`: You set global configurations for the Astro CLI by running `astro config --global`. See [Configure the CLI](configure-cli.md#global-configuration-options) for more information.
```
